Robot2018/src/org/usfirst/frc/team199/Robot2018/commands/RunScript.java

```diff
@@ -1,38 +1,34 @@
 package org.usfirst.frc.team199.Robot2018.commands;
 
 import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Map;
 
 import org.usfirst.frc.team199.Robot2018.Robot;
 
-import edu.wpi.first.wpilibj.Timer;
 import edu.wpi.first.wpilibj.command.CommandGroup;
 import edu.wpi.first.wpilibj.command.WaitCommand;
 
 /**
- * Gets the script name and runs the script.
+ * Gets the script name and interprets the script into individual commands.
  */
 public class RunScript extends CommandGroup implements RunScriptInterface {
```

Comment:
This class does additional parsing and it also has at least one bug. Needs tests.